build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 import java.util.Base64
 
 allprojects {
-    version = "0.0.5"
+    version = "1.0.0-SNAPSHOT"
```

Comment:
Hold your horses! :horse_racing:

Reply:
Constructor of `OpaFilter` has changed. Whoever used it without a starter, now has a breaking change. According to SemVer, this has to be 1.0.0.

@anderseknert ?